scripts/plugins/install_plugins.py

```diff
@@ -183,7 +180,7 @@ def Install_Plugins():
                             doc_link=Get_Doc("missing_plugin_dependency"),
                         )
                     )
-                    exit(10)
+                    sys.exit(10)
```

Comment:
Please don't change something that's outside of this PR scope.
Let's add `exit` -> `sys.exit` only if there's a new logic introduced in this PR.

Everything else is already here: https://github.com/bitovi/bitops/pull/298/commits/7b7f9b9aa2f94b9665f924896388ec04abb89fca